sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1169,9 +1169,11 @@ def delete_blob(
         blob_client = self.get_blob_client(blob) # type: ignore
         kwargs.setdefault('merge_span', True)
         timeout = kwargs.pop('timeout', None)
+        version_id = kwargs.pop('version_id', None)
```

Comment:
For all container clients methods (sync and async) I believe this can just pass through `kwargs` now? No need to pop it and pass it. I think this was left over from when we needed more logic in the ContainerClient.